libbeat/asset/asset.go

```diff
@@ -0,0 +1,49 @@
+package asset
+
+import "html/template"
```

Comment:
This was using `text/template`.

Reply:
You just saved my day. I already started to smash my head against the wall on why there is a diff in the fields.go content but there shouldn't be one. I definitively owe you some beers. 🍻